src/apiCalls.js

```diff
@@ -13,4 +14,4 @@ const fetchData = (url) => { return fetch(url).then(response => {
     })
 }
 
```

Comment:
I feel like we could pull the (!response.ok) logic out into a reusable function so we can use it in both our GET and POST requests